starter-code/.gitignore

```diff
@@ -0,0 +1,21 @@
+# See https://help.github.com/ignore-files/ for more about ignoring files.
```

Comment:
(I believe) you can also add the .gitignore file to the actual ignore files this way she doesn't show up on GH